src/Mesh/DerivedData.C

```diff
@@ -1036,20 +1121,25 @@ std::vector< std::size_t >
 genInpofaTet( std::size_t ntfac,
               std::size_t nbfac,
               const std::vector< std::size_t >& inpoel,
+              const std::vector< std::size_t >& triinpoel,
               const std::vector< int >& esuelTet )
 // *****************************************************************************
 //  Generate derived data, points on faces for tetrahedra only
 //! \param[in] ntfac Total number of faces.
 //! \param[in] nbfac Number of boundary faces.
 //! \param[in] inpoel Element-node connectivity.
+//! \param[in] triinpoel Face-node connectivity.
 //! \param[in] esuelTet Elements surrounding elements.
 //! \return Elements surrounding faces.
-//! \details The unsigned integer map gives the elements to the left and to
+//! \details The unsigned integer vector gives the elements to the left and to
 //     the right of each face in the mesh.
```

Comment:
This `\details` comment really relates to the return vector, correct? If so, let's just move it to after the description of `\return`.